### PR TITLE
005 competencies per band

### PR DESCRIPTION
**Pull request by mactrz**
As a Kainos employee
I want to find a list of Kainos Competencies expected of an employee within each band level
So that I know the competencies expected of an employee performing at that band level

Tech notes:
- UI page to display results from API

### CODE REVIEW

File: views/bandCompetency.njk [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{% extends "./templates/template.njk" %}
+
+{% block content %}
+<form onsubmit="location.href='http://www.localhost:8000/band-competency/' + document.getElementById('select-band').value; return false;" style="margin-top: 10px">
+    <select id="select-band" class="inputGroupSelect04">
+        <option value="">Pick a band level</option>
+        <option value="Apprentice">Apprentice</option>
```
[the remark this answers is not in the document]
Comment (mactrz) [reply views/bandCompetency.njk]: Bands now fetched from backend

**Review by michaltangri — CHANGES_REQUESTED** [review]
Could you remove the `console.log(personalPerformance)` from **util/bandCompetencyUtil.js** please?

**Review by michaltangri — APPROVED** [review]
Looks good to me

File: views/templates/template.njk [hunk 2]
```diff
@@ -31,9 +31,12 @@
                                 <li class="nav-item">
 									<a class="nav-link" href="/capabilities">Capability</a>
 								</li>
+                <li>
```
Comment (sebastianczajkowski) [inline views/templates/template.njk:34]: `<li>` is missing `class="nav-item"`

**Review by sebastianczajkowski — CHANGES_REQUESTED** [review]
Two `<li>` tags need a class

File: views/templates/template.njk (hunk 2 above)
Comment (mactrz) [reply views/templates/template.njk]: Added the class tag

**Review by sebastianczajkowski — APPROVED** [review]
Good stuff

**Review by TheoHodges — APPROVED** [review]

**Pull request merged**